gen/ad_user_mu_ucn

```diff
@@ -113,6 +133,10 @@ for my $login (@logins) {
 	if (defined $mail and length $mail) {
 		print FILE "mail: " . $mail . "\n";
 	}
+	foreach (@{$users->{$login}->{$A_CHIPNUMBERS}}){
+		next if ($_ eq "");
+		print FILE "houseIdentifier: " . checkBase64($_) . "\n";
```

Comment:
You shouldn't have ending with space, since you optionally append ``:`` and base64 value, so you end up with ``houseIdentifier: :somevalue``, which I think is not valid. Spaces between separator ``: or ::`` and value are optional, so you can discard it in this not and don't have to implement any additional logic.